lib/rubysl/rubysl-strscan/lib/rubysl/strscan/strscan.rb

```diff
@@ -24,6 +24,17 @@
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
 # EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+# Modifications made by the Truffle team are:
```

Comment:
`... the Truffle team ...` shouldn't it be TruffleRuby?

Reply:
I just copied the header added to other files we modified. Should it be changed now we are truffleruby?

Reply:
Ah we should probably do that in another commit then, sorry did not notice it before.